server/server.js

```diff
@@ -13,15 +14,22 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 
 app.get('/api/updates/:id', (req, res) => {
-  db.loadProject(req.params.id, (err, project) => {
+  cacheGetProject(req.params.id, (err, project) => {
     if (err) {
       console.log('ERROR LOADING PROJECT', err);
+      res.status('400')
+      res.send(err)
     } else {
-      res.json(project);
+      project = (typeof project === 'string') ? project : JSON.stringify(project); 
```

Comment:
When is the conditional in this line relevant?

Reply:
redis returns a string, but mongo returns an object, so we want to send consistent data to the user by only stringifying if we got the data from redis. I could potentially refactor this so that it is internal to the function call